pants.toml

```diff
@@ -26,6 +26,7 @@ backend_packages.add = [
   "pants.backend.experimental.java.debug_goals",
   "pants.backend.experimental.python",
   "pants.backend.experimental.python.packaging.pyoxidizer",
+  "pants.backend.experimental.python.lint.ruff",
```

Comment:
@thejcannon Please lmk if we don't want to include ruff in the toml file yet

Reply:
Removed (for now) since ruff is complaining for some files

Reply:
Yeah let's add this in its own PR